lib/v2/screens/explore_screens/vote_screens/vote/vote_screen.dart

```diff
@@ -21,12 +22,13 @@ class VoteScreen extends StatelessWidget {
             title: Text('Vote'.i18n),
             bottom: TabBar(
                 labelPadding: const EdgeInsets.all(8.0),
+                indicatorColor: AppColors.green1,
```

Comment:
is this something we should do globally? in the theme?

Reply:
mm.. this is strange I saw you bro remove this